src/bun.zig

```diff
@@ -514,6 +514,10 @@ pub fn ensureNonBlocking(fd: anytype) void {
     _ = std.os.fcntl(fd, std.os.F.SETFL, current | std.os.O.NONBLOCK) catch 0;
 }
 
+pub fn isNonBlocking(fd: anytype) bool {
+    return (std.os.fcntl(fd, std.os.F.GETFL, 0) catch 0 & std.os.O.NONBLOCK) == std.os.O.NONBLOCK;
```

Comment:
```suggestion
    if (Environment.isWindows) {
        @compileError("isNonBlocking is not supported on windows");
    }
    return (std.os.fcntl(fd, std.os.F.GETFL, 0) catch 0 & std.os.O.NONBLOCK) == std.os.O.NONBLOCK;
```

Reply:
Thanks for looking at this, do you need any action from me? Would you like me to rebase against master and apply the above suggestion?

Reply:
yeah that would be great 👍

Reply:
Done, I tried to align the formatting to that of other @compileError functions within the file. I also squashed the formatting commit into the original commit to keep the git history clean.